helm/charts/infra/templates/server/secret.yaml

```diff
@@ -0,0 +1,29 @@
+
+{{- if include "server.enabled" . | eq "true" }}
+{{- if (not .Values.server.config.tls) }}
+{{- $secret := lookup "v1" "Secret" .Release.Namespace (printf "%s-ca" (include "server.fullname" .)) -}}
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "server.fullname" . }}-ca
+  labels:
+{{- include "server.labels" . | nindent 4 }}
+stringData:
+
+{{- if $secret.data }}
+  ca.crt: |
+{{- get $secret.data "ca.crt" | b64dec | nindent 4 }}
+  ca.key: |
+{{- get $secret.data "ca.key" | b64dec | nindent 4 }}
+
+{{- else }}
+{{- $ca := genCA "Infra Server" 3650 }}
```

Comment:
This is the name people will see in the Subject line of the certificate. Should we use "Infra" , or "Infra Server" ?

Reply:
"Infra Server" seems fine to me 👍